### ISSUE TITLE
Wrong weekday in CronTrigger 

### ISSUE THREAD

**Issue by hmiguel** [agronholm/apscheduler#411]
## Expected Behavior

The weekday value '0' should be Sunday.

## Current Behavior

When the weekday value is set to 0, the trigger assumes Monday.

## Steps to Reproduce

```
def my_cron_job():
    print ("executed.")

if __name__ == '__main__':
    from apscheduler.schedulers.background import BackgroundScheduler
    from apscheduler.triggers.cron import CronTrigger
    
    sched = BackgroundScheduler(timezone='WET')

    sched.add_job(my_cron_job, id='job_weekday_0', trigger = CronTrigger.from_crontab('0 2 * * 0')) 

    sched.add_job(my_cron_job, id='job_weekday_SUN', trigger = CronTrigger.from_crontab('0 2 * * SUN')) 
    
    sched.start()
    
    j = sched.get_job('job_weekday_0')
    
    print (j.id, j.next_run_time) # job_weekday_0 2019-12-23 02:00:00+00:00

    j = sched.get_job('job_weekday_SUN')
    
    print (j.id, j.next_run_time) # job_weekday_SUN 2019-12-22 02:00:00+00:00

````

**Comment by agronholm** [comment]
Duplicate of #286?

**Comment by kaos** [comment]
Also proposed fix in #381

**Comment by agronholm** [comment]
Closing as duplicate.
